lib/view_model/active_record/nested_controller_base.rb

```diff
@@ -78,22 +151,16 @@ def owner_viewmodel_id(required: true)
     parse_param(id_param_name, **default)
   end
 
-  included do
-    delegate :owner_viewmodel, :association_name, to: 'self.class'
+  def owner_viewmodel_class_for_name(name)
```

Comment:
Is this indirection here as a test injection hook?

Reply:
This is intended to be an escape hatch. The route parameters are defined as strings, but I didn't want to force consumers to be tightly bound to the name registry. That said, I'm struggling to see a motivating use case.